some changes

cuttazy opened the pull request:
- get courses list only one time (unique query) 
- removed remote courses function (adapted for institutions with 1 moodle)
- adapted overviews only for the favourites (less time of execution)

- Merge changes "Add Privacy API support and some small css fixes"

cuttazy committed:
- get courses list only one time (unique query)
- removed remote courses function (adapted for institutions with 1 moodle)
- adapted overviews only for the favourites (less time of execution)

cuttazy committed:
merge changes "Add Privacy API support and some small css fixes" from thepurpleblob

cuttazy committed:
changed col-sm to col-md in layout

cuttazy committed:
changed text in "nocourse" label

thepurpleblob commented:
Thanks.

Just at a glance... there's quite a few commented out lines and it should be "your" not "you" in that text string. 

I'll have a proper look when I have a bit more time!

cuttazy committed:
- has overview check: print col-md-12 if hasn't overviews

cuttazy commented:
Yes, I'm still working on. I want to show how to fix deprecated functions too. Thank's for your feedback

cuttazy committed:
error messages different in every tab

cuttazy committed:
Delete .DS_Store

cuttazy committed:
added cursor pointer

cuttazy committed:
- resolved bug sort favourites
- cursor to drag&drop on elements

cuttazy committed:
- Overview in courses list ON/OFF with setting for admin
- Background color for courses with updates with admin setting
- Translations IT, FR, DE
- bug reorder drag&drop in favourites

cuttazy commented:
- Overview in courses list ON/OFF with setting for admin
- Background color for courses with updates with admin setting
- Translations IT, FR, DE
- bug reorder drag&drop in favourites

cuttazy committed:
- added classes in moustach
- changed style of close button in popup

cuttazy committed:
- Dialog overlay
- Default tab setting

cuttazy committed:
overlay bug resolved

thepurpleblob commented:
Sorry - I'm getting a bit lost with this. I've sorted a few of these things myself and we've moved on a version or two. Any chance of a rebase/update?

cuttazy committed:
dialog style

cuttazy committed:
close button dialog

cuttazy committed:
fix margin close button

cuttazy committed:
min-width popup

cuttazy committed:
href not deleted on edit mode

cuttazy committed:
dialog enabled when editing

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
Changes for local_custom_notification plugin.
Dashboard have now notification subrscribtion for each course

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

thepurpleblob commented:
Getting loads of what looks like development stuff pushed to this. Not sure if that's what you intended.

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
temp debug push

Commit:
string translations

Commit:
version 3.6.0

Commit:
dimmed text color fix

Commit:
added visible = true to recently accessed courses

Commit:
disabled overviews

Commit:
deleted overview functions

Commit:
tolto jquery ui

Commit:
jquery ui in edit
